nss-day-cohort-29/welcome-to-nashville-valkyries
# Initial code to return concerts search results.

# Description

Initial code to search Ticketmaster API using stored variable from user input (concertsSearchParam), the current date from 00:00:01 to 11:59:59 and a location of Nashville, TN.


## Type of change

New code.  Currently the code uses a fixed search term (concertsSearchParam = "music") that will be replaced by the variable captured from the user input.  The code returns an array called parsedConcerts.  This variable will be passed into the domCompBuilder function via that last line of code, which is currently commented out.  Also, the call of concertsFetch() is temporary, once the eventListeners.js file is complete, the event listener will execute this call and this line will be removed.

# Testing Instructions for Change Made

1. git fetch --all
2. git checkout rr-concertscripts
3. run grunt from lib directory
4. in your local server (chrome browser) you should see a console log of the returned results array


# Checklist:
